lib/renderer/api/remote.js

```diff
@@ -175,6 +175,7 @@ const proxyFunctionProperties = function (remoteMemberFunction, metaId, name) {
       return true
     },
     get: (target, property, receiver) => {
+      if (property === 'toString') return target.toString.bind(target)
```

Comment:
@twolfson The case for `toString` being not the built-in function is quite easy to reproduce.

```js
var a = new Proxy({ toString: 'foo' }, { get: (a, b) => a[b] })
```

In this case `a.toString` returns the string "foo".  I think the safest check to do is this.

```js
if (property === 'toString' && target.constructor.prototype.toString === target.toString) {
  // Return bind
}
```

This ensures that not only is the toString property a function but it is the original built-in function that the object was created with.